src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts

```diff
@@ -1244,7 +1246,7 @@ export class GettingStartedPage extends EditorPane {
 		super.clearInput();
 	}
 
-	private buildCategorySlide(categoryID: string, selectedStep?: string) {
+	private buildCategorySlide(categoryID: string, selectedStep?: string, telemetryFooter?: boolean) {
```

Comment:
You are not using the `telemetryFooter` flag anywhere.

Reply:
Right, my bad. Was just for testing to make sure it always shows up.